### ISSUE TITLE
duoCo strip app devices compatibility

### ISSUE THREAD

**Issue by gchrisak** [sysofwan/ha-triones#10]
Hello, would it be possible to make this compatible with the Bluetooth devices (ELK-BLEDOM) working with the duoCo strip app? The app is exactly the same as the triones app.
Maybe this can help:
https://github.com/TheSylex/ELK-BLEDOM-bluetooth-led-strip-controller

**Comment by wroblear** [comment]
Hi @gchrisak Im using the same device, and cant tun it, 
Can't add Triones integration, its showes me that I cant add device, 
Autodiscovery doesnt work

**Comment by gchrisak** [comment]
Yeah ha-trones is not compatible with this device. That's why I made this thread.
 Its a very popular device with over 2000 sales on aliexpress. Hopefully someone can make it work with Homeassistant

**Comment by sysofwan** [comment]
Unfortunately, I do not have the hardware to test this. The API looks significantly different that it probably requires another implementation.

Would appreciate contributions though 😄

**Comment by wroblear** [comment]
I can sent you such a hardware if you want and have time to setup it, 
strange is that i can see in logs:
![image](https://user-images.githubusercontent.com/90571492/163669874-044a52d9-dd26-448e-b388-d90cba8a1da7.png)
but when Im trying to connect it,  I getting: unnable to connect.
How I can force 'autodiscovery' option ?

**Comment by gchrisak** [comment]
Seems like a lot of wotrk has been done here too.
https://github.com/FergusInLondon/ELK-BLEDOM
https://github.com/FergusInLondon/ELK-BLEDOM/blob/master/PROTCOL.md

Unfortunatly i dont have the knowledge to do this my self.

**Comment by dave-code-ruiz** [comment]
I create repository for this strip in HACS:

https://github.com/dave-code-ruiz/elkbledom-ha

But now i have a problem with read status strip values, today code is under construction

thnks

**Comment by gchrisak** [comment]
That's great news! Thank you very much!

**Comment by dave-code-ruiz** [comment]
Now strip works with some limitations

**Comment by gchrisak** [comment]
Confirmed!
